NuRadioReco/framework/base_trace.py

```diff
@@ -124,6 +124,11 @@ def get_number_of_samples(self):
             length = (self._frequency_spectrum.shape[-1] - 1) * 2
         return length
 
+    def apply_time_shift(self, delta_t):
```

Comment:
Can you add a description including the warning that this is a cyclic shift? We might even print a warning if deltaT is a significant fraction of the trace length

Reply:
Done